.eslintrc.js

```diff
@@ -128,10 +128,17 @@ module.exports = {
     'unicorn/filename-case': 'off',
     'unicorn/no-null': 'off',
     'unicorn/no-array-callback-reference': 'off', // doesn't work well with TypeScript's custom type guards
+    'unicorn/no-negated-condition': 'off', // turn off to prevent make breaking changes (ref: #2146)
     'unicorn/no-useless-undefined': 'off',
     'unicorn/prefer-optional-catch-binding': 'off', // not supported by current ESLint parser version
     'unicorn/prefer-module': 'off',
     'unicorn/prevent-abbreviations': 'off',
+    'unicorn/prefer-at': 'off', //                 turn off to prevent make breaking changes (ref: #2146)
+    'unicorn/prefer-node-protocol': 'off', //      turn off to prevent make breaking changes (ref: #2146)
+    'unicorn/prefer-string-replace-all': 'off', // turn off to prevent make breaking changes (ref: #2146)
+    'unicorn/prefer-ternary': 'off', //            turn off to prevent make breaking changes (ref: #2146)
+    'unicorn/prefer-top-level-await': 'off', //    turn off to prevent make breaking changes (ref: #2146)
+    'unicorn/switch-case-braces': 'off', //        turn off to prevent make breaking changes (ref: #2146)
```

Comment:
`unicorn/no-negated-condition`, `unicorn/prefer-ternary` and `unicorn/switch-case-braces` don't cause breaking changes, so I think they should be enabled (and the respective lint errors be fixed).

Reply:
I don't want that rule enabled. because it conflicts with the current open pull request. I think it's better to enable the rule when there are few pull requests.

Reply:
for now, should I change comments only?

`turn off to prevent make breaking changes (ref: #2146)`
=> `remove when there are few pull requests (ref: #2146)`

Reply:
Yes, I think that will be a good compromise :+1:

Reply:
@FloEdelmann 
Done!  6f4dca6